src/presentation/components/Settings/AxisSetSettings.vue

```diff
@@ -405,6 +386,11 @@ export default defineComponent({
         font-size: 0.75rem;
       }
     }
+
```

Comment:
It isn't part of the current changes, but is box-shadow: 2px; considered a mismatched property value?